### PR TITLE
update side landing menu with BA List

### PR DESCRIPTION
**Pull request by ladleo**
#257

### CODE REVIEW

File: opensrp-chw/src/main/java/org/smartregister/chw/model/NavigationModel.java [hunk 1]
```diff
@@ -21,8 +21,14 @@
     private NavigationModel() {
         navigationOptions.clear();
         NavigationOption op1 = new NavigationOption(R.mipmap.sidemenu_families, R.mipmap.sidemenu_families_active, Constants.DrawerMenu.ALL_FAMILIES, 0);
-        NavigationOption op2 = new NavigationOption(R.mipmap.sidemenu_children, R.mipmap.sidemenu_children_active, Constants.DrawerMenu.CHILD_CLIENTS, 0);
-        navigationOptions.addAll(asList(op1, op2));
+        NavigationOption op2 = new NavigationOption(R.mipmap.sidemenu_anc, R.mipmap.sidemenu_anc_active, Constants.DrawerMenu.ANC, 0);
```
Comment (rkodev) [inline opensrp-chw/src/main/java/org/smartregister/chw/model/NavigationModel.java]: @ladleo do not overwrite the changes form the default chw app. You use conditional logic to check the version of the app and only modify for the BA/TZ build

File: opensrp-chw/src/main/java/org/smartregister/chw/listener/NavigationListener.java [hunk 2]
```diff
@@ -33,6 +34,24 @@ public void onClick(View v) {
                     case Constants.DrawerMenu.ALL_FAMILIES:
                         startRegisterActivity(FamilyRegisterActivity.class);
                         break;
```
Comment (rkodev) [inline opensrp-chw/src/main/java/org/smartregister/chw/listener/NavigationListener.java]: @ladleo make these changes only available to TZ build of the app

File: opensrp-chw/src/main/java/org/smartregister/chw/provider/FamilyActivityRegisterProvider.java [hunk 3]
```diff
@@ -64,6 +64,7 @@ protected void populatePatientColumn(CommonPersonObjectClient pc, SmartRegisterC
         }
 
         String patientName = getName(firstName, middleName, lastName);
+//        String d
```
Comment (rkodev) [inline opensrp-chw/src/main/java/org/smartregister/chw/provider/FamilyActivityRegisterProvider.java]: whats this for?

File: opensrp-chw/src/main/java/org/smartregister/chw/model/NavigationModel.java (hunk 1 above)
Comment (ladleo) [reply opensrp-chw/src/main/java/org/smartregister/chw/model/NavigationModel.java]: Updated it

File: opensrp-chw/src/main/java/org/smartregister/chw/provider/FamilyActivityRegisterProvider.java (hunk 3 above)
Comment (ladleo) [reply opensrp-chw/src/main/java/org/smartregister/chw/provider/FamilyActivityRegisterProvider.java]: Removed

**Pull request merged**